view/frontend/templates/mailchimpjs.phtml

```diff
@@ -1,4 +1,4 @@
 <?php
-    $url = $this->getJsUrl();
+    $url = $block->escapeHtml($this->getJsUrl());
```

Comment:
Actually escaping better to do just during printing variable, so it's better to add this code few lines below